### ISSUE TITLE
Embedded tabs cannot share `role: tab`, must use `role: link`, which breaks accessibility

### ISSUE THREAD

**Issue by bwmcdonough** [reactjs/react-tabs#309]
Expected behavior: When two sets of tabs exist on the same page with one set of tabs wrapping the other, setting `role: tab` on both tab sets should not result in the inner tab set controlling the outer tab set.

Actual behavior: The inner set of tabs must use `role: link` in order to avoid the inner set of tabs controlling the outer set of tabs.

For accessibility, the ability to assign `role: tab` to both sets is strongly preferred.

**Comment by brianebill** [comment]
In diving deeper into the root cause, this may be related to a missing connection in our implementation of the API. I'll circle back here after I take a deeper dive.
